crates/icrate/Cargo.toml

```diff
@@ -85,7 +85,10 @@ required-features = [
 ]
 
 [features]
-default = ["std", "apple"]
+default = ["std", "apple", "link"]
+
+# Link against framework, otherwise the correct framework should be load at runtime
```

Comment:
Nit: Can you document the use-case you mentioned, as well as the related use-case in https://github.com/servo/core-foundation-rs/pull/608?